src/t_hash.c

```diff
@@ -117,6 +124,14 @@ void hashTypeTrackUpdateEntry(robj *o, void *old_entry, void *new_entry, long lo
     }
 }
 
+static inline void debugLogField(robj *key, void *entry) {
+    if (1 <= LL_VERBOSE) {
```

Comment:
Ignore this for the time being. I'll eventually remove this function altogether. Its used for development/debugging.